docs/release-notes/master.adoc

```diff
@@ -13,10 +13,46 @@ include::topics/templates/document-attributes.adoc[]
 //Inclusive language statement
 include::topics/making-open-source-more-inclusive.adoc[]
 
-include::topics/mta-rn-new-features.adoc[leveloffset=+1]
+[id="release-notes-introduction"]
+== Introduction
+{ProductName} {DocInfoProductNumber} accelerates large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift via the user interface.
+
+These release notes cover all _z_-stream releases of {ProductShortName} 6.0 with the most recent release listed first.
+
+[id="mta-6-0-1"]
+== {ProductShortName} 6.0.1
+
+include::topics/mta-rn-new-features-1.adoc[leveloffset=+2]
+
+You can set this parameter after you upgrade to {ProductVersion}.
```

Comment:
@RichardHoch maybe you can make it a note?
Note: If you upgrade from version 6.0.0 to version 6.0.1, you can set this parameter after the upgrade